WasatchNET/Spectrometer.cs

```diff
@@ -2327,25 +2439,28 @@ protected void correctBadPixels(ref double[] spectrum)
         /// <returns>The acquired spectrum as an array of doubles</returns>
         public virtual double[] getSpectrum(bool forceNew=false)
         {
+            var driver = Driver.getInstance();
             lock (acquisitionLock)
             {
+                uptime.setError(uniqueKey); // assume acquisition may fail

```

Comment:
Even though this makes sense as a flag it still feels weird when an API to me for some reason (sorry for rambling)

Reply:
I understand.  Perhaps "notifyErrorDetected" or something?